### ISSUE TITLE
Cannot login windows 7

### ISSUE THREAD

**Issue by zotix** [synergy/synergy#4980]
Server: Windows 7
Client: Mac 10.10.4

When windows 7 logout, I cannot typo "u" because it always show up accessibility window.

**Comment by VeNoMouS** [comment]
sounds like stuck key behaviour

**Comment by awollangk** [comment]
Ctrl-U is how you bring up the accessibility so if just U brings it up your Ctrl key is stuck somehow.
